EU4toV2/Source/V2World/V2World.cpp

```diff
@@ -2105,4 +2116,61 @@ void V2::World::updateCountryDetails()
 			}
 		}
 	}
+}
+
+void V2::World::processShatteredHre(const std::optional<std::string>& eu4HreTag)
+{
+	if (!eu4HreTag)
+		return;
+
+	const auto& v2HreTag = countryMapper.getV2Tag(*eu4HreTag);
```

Comment:
if you already used countryMapper, no need to use it again. it's optional so if it's not present you can skip.